INFO     conftest:conftest.py:29 mysql conenction is getting established
INFO     conftest:conftest.py:32 mysql conenction has been established
INFO     conftest:conftest.py:39 mysql conenction is getting established
INFO     conftest:conftest.py:42 mysql conenction has been established
INFO     TestScripts.test_Data_Loading:test_Data_Loading.py:55 Test case execution for inventory_level_stores_check check has started....
INFO     CommonUtils.utils:utils.py:54 The expected data in the file is :   store_id  total_inventory
0         1            115.0
1         2            195.0
2         3            265.0
3         4            220.0
4         5            190.0
5         6            260.0
6         7            230.0
7         8            270.0
8         9            320.0
9        10            170.0
INFO     CommonUtils.utils:utils.py:56 The actual data in the file is :   store_id  total_inventory
0         1            115.0
1         2            195.0
2         3            265.0
3         4            220.0
4         5            190.0
5         6            260.0
6         7            230.0
7         8            270.0
8         9            320.0
9        10            170.0
INFO     TestScripts.test_Data_Loading:test_Data_Loading.py:63 Test case execution for inventory_level_stores_check has completed....
INFO     conftest:conftest.py:45 mysql conenction has been closed
INFO     conftest:conftest.py:35 mysql conenction has been closed
